packages/nimble-components/src/anchor-menu-item/index.ts

```diff
@@ -0,0 +1,106 @@
+import { attr, observable } from '@microsoft/fast-element';
+import {
+    DesignSystem,
+    AnchorOptions,
+    MenuItem as FoundationMenuItem
+} from '@microsoft/fast-foundation';
+import { keyEnter } from '@microsoft/fast-web-utilities';
+import { AnchorBase } from '../anchor-base';
+import { styles } from './styles';
+import { template } from './template';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-anchor-menu-item': AnchorMenuItem;
+    }
+}
+
+/**
+ * Types of anchor menu item column count.
```

Comment:
nit: I don't think it's immediately obvious what a column count is on a menu item. Indentation level? Number of child elements? Could you add more detailed documentation of this type and/or its corresponding property? Are clients expected to set the property?

Reply:
Done